editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/FusePropertySection.java

```diff
@@ -432,7 +432,9 @@ protected void languageChanged(String language, Composite eform, AbstractCamelMo
 				this.selectedEP.setParameter(prop.getName(), expContainerElement);
 				uiExpressionElement = expressionElement;
 
-				updateDependencies(lang.getDependencies());
+				if (lang != null) { // some languages are not defined in catalog like "method"
```

Comment:
I think method is not a real language. That's the whole point. But yeah...probably Camel should do something about that in terms of Catalog completeness. @davsclaus